File: 69. Sqrt(x)/solution_1.js
/**
 * @param {number} x
 * @return {number}
 */


var mySqrt = function(x) 
{
    let sqrt = 0

    for (let i = 0; i <= x; ++i)
    {
        if (i * i <= x)
            sqrt = i
        else
            break
    }   

    return sqrt 
};

File: 0069. Sqrt(x)/solution_1.cs
public class Solution
{
    public int MySqrt(int x)
    {
        long sqrt = 0;

        for (long i = 0; i <= x; ++i)
        {
            if (i * i <= x)
                sqrt = i;
            else
                break;
        }

        return (int)sqrt;
    }
}